packages/koa-metrics/src/index.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable camelcase */
+import {StatsD} from 'hot-shots';
+import {Context} from 'koa';
+
+export interface Options {
+  prefix: string;
+  host: string;
+}
+
+export default function metrics({prefix, host}: Options) {
+  const statsdHost = host.split(':')[0];
+  const statsdPort = parseInt(host.split(':')[1], 10);
+
+  return async function statsdMiddleware(ctx: Context, next: Function) {
+    const requestStartTime = process.hrtime();
+    // @ts-ignore the types here are correct. TS complains about method
```

Comment:
What method is potentially undefined? How can it be true that the types are correct and also we need this @ts-ignore?

Can w add a fix for this type definition to our `config/typescript` and open a PR on definitely typed instead?

Reply:
I don't get it either, I'd rather one cast than losing strong typing on the entire statement